apps/videos-admin/src/app/(dashboard)/videos/[videoId]/citation/_CitationForm/CitationForm.tsx

```
@@ -0,0 +1,364 @@
+              verseStart: '',
+              verseEnd: ''
+            }
+      }
+      onSubmit={handleSubmit}
+      validationSchema={validationSchema}
+    >
+      {({ values, errors, handleChange, isValid, isSubmitting, dirty }) => (
+        <Form>
+          <Stack gap={3}>
+            <FormControl fullWidth>
+              <InputLabel id="bibleBookId-label">Bible Book</InputLabel>
```

ESLint

[error] disallow literal string: <InputLabel id="bibleBookId-label">Bible Book</InputLabel>

(i18next/no-literal-string)